library/src/main/java/com/paypal/messages/logger/Logger.kt

```diff
@@ -74,13 +78,27 @@ class Logger private constructor() {
 	}
 
 	fun setGlobalAnalytics(
```

Comment:
Did you decided against adding setGlobalAnalytics to the companion object ?

Reply:
Fixed. Created a GlobalAnalytics object that will be used for static purposes